src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let carCosts = 0;
+  const dailyRent = 40;
+
+  if (days >= 7) {
+    carCosts = days * dailyRent - 50;
+  } else if (days >= 3) {
+    carCosts = days * dailyRent - 20;
```

Comment:
50 and 20 are the magic numbers, create a variables for them, like you did for `dailyRent`